api/maven-api-core/src/main/java/org/apache/maven/api/Project.java

```diff
@@ -37,24 +37,58 @@
 @Experimental
 public interface Project {
 
+    /**
+     * Returns the project groupId.
+     */
     @Nonnull
     String getGroupId();
 
+    /**
+     * Returns the project artifactId.
+     */
     @Nonnull
     String getArtifactId();
 
+    /**
+     * Returns the project version.
+     */
     @Nonnull
     String getVersion();
 
+    /**
+     * Returns the project packaging.
+     */
     @Nonnull
     String getPackaging();
 
+    /**
+     * Returns the project artifact, that is the artifact produced by this project. If the artifact file exists, it is
+     * registered to {@link org.apache.maven.api.services.ArtifactManager}.
+     *
+     * @see org.apache.maven.api.services.ArtifactManager#getPath(Artifact)
+     */
     @Nonnull
     Artifact getArtifact();
 
+    /**
+     * Returns the project artifacts, that is, the project POM artifact and the artifact produced by this project.
+     * The list may have 1 or 2 elements (never less than 1, never more than 2), depending on project packaging. If
+     * artifact files exists, they are registered to {@link org.apache.maven.api.services.ArtifactManager}.
+     *
+     * @see org.apache.maven.api.services.ArtifactManager#getPath(Artifact)
+     */
+    @Nonnull
+    List<Artifact> getArtifacts();
```

Comment:
Will this include the artifact from above?

Reply:
yes